addGame.php

```diff
@@ -13,32 +13,32 @@
 <body>
     <nav>
         <div class="navHexContainer">
-            <div class="navHexRow first">
-                <div class="navHex">
-                    <img src="home.png" alt="home icon">
-                </div>
-                <div class="navHex">
-                    <img src="folder.png" alt="folder icon">
-                </div>
-               <div class="navHex">
-                    <img src="user.png" alt="user icon">
-                </div>
-            </div>
+          <div class="navHexRow first">
+             <a class="navHex" href="index.php">
+                 <img src="home.png" alt="home icon">
+             </a>
+             <a class="navHex" href="index.php">
+                <img src="folder.png" alt="folder icon">
+             </a>
+             <a class="navHex" href="#">
+                <img src="user.png" alt="user icon">
+             </a>
+          </div>
             <div class="navHexRow last">
-                <div class="navHex">
-                    <img src="friends.png" alt="group icon">
-               </div>
-                <div class="navHex">
-                  <img src="shop.png" alt="online shop icon">
-                </div>
+                <a class="navHex" href="#">
+                 <img src="friends.png" alt="group icon">
+              </a>
+                <a class="navHex" href="https://www.amazon.co.uk/s?k=board+games&sprefix=boar%2Caps%2C64&ref=nb_sb_ss_ts-doa-p_1_4" target="_blank">
+                    <img src="shop.png" alt="online shop icon">
```

Comment:
image is quite difficult to see on a smaller screen (but this is a very pedantic comment so feel free to ignore)